libraries/AP_HAL_PX4/RCInput.cpp

```diff
@@ -111,6 +119,30 @@ void PX4RCInput::clear_overrides()
     }
 }
 
+const char *PX4RCInput::input_source_name(uint8_t id) const
+{
+    switch(id) {
+    case input_rc_s::RC_INPUT_SOURCE_UNKNOWN:         return "UNKNOWN";
+    case input_rc_s::RC_INPUT_SOURCE_PX4FMU_PPM:      return "PX4FMU_PPM";
```

Comment:
@peterbarker I confirmed that it is in the order of auto genarated file "ardupilot/modules/PX4Firmware/src/modules/uORB/topics/input_rc.h".